class.jetpack-cli.php

```diff
@@ -291,24 +310,79 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::line( __( "Resetting default modules...\n", "jetpack" ) );
 				usleep( 500000 ); // Take a breath
 				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					Jetpack::update_active_modules( $default_modules );
+				}
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 
 				// Jumpstart option is special
-				Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				if ( ! $is_dry_run ) {
+					Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				}
 				WP_CLI::success( __( 'jumpstart option reset', 'jetpack' ) );
 				break;
 			case 'modules':
-				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					$default_modules = Jetpack::get_default_modules();
+					Jetpack::update_active_modules( $default_modules );
+				}
+
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites        = get_sites( array( 'number' => 1000 ) );
+					$count_fixes  = 0;
+					foreach ( $sites as $site ) {
+						switch_to_blog( $site->blog_id );
+						$count = self::count_option( $option );
+						if ( $count > 1 ) {
+							if ( ! $is_dry_run ) {
+								delete_option( $option );
+							}
+							WP_CLI::line( sprintf( __( 'Deleted %s %s options from %s', 'jetpack' ), $count, $option, "{$site->domain}{$site->path}" ) );
+							$count_fixes++;
+							$sleep_duration = ( $is_dry_run ? 1 : 3 );
+							sleep( $sleep_duration ); // Allow some time for replication to catch up.
+						}
+
+						restore_current_blog();
+					}
+					if ( $count_fixes ) {
+						WP_CLI::success( sprintf( __( "Successfully reset %s on %s sites.", 'jetpack' ), $option, $count_fixes ) );
+					} else {
+						WP_CLI::success( __( "No options were deleted.", 'jetpack' ) );
+					}
+					return;
+				}
+
+				$count = self::count_option( $option );
+				if ( $count > 1 ) {
+					if ( ! $is_dry_run ) {
+						delete_option( $option );
+					}
+					WP_CLI::success( sprintf( __( 'Deleted %s %s options', 'jetpack' ), $count, $option ) );
```

Comment:
Perhaps splitting those calls into lines is a good idea?

```suggestion
					WP_CLI::success(
						sprintf(
							__( 'Deleted %s %s options', 'jetpack' ),
							$count,
							$option
						)
					);
```